### PR TITLE
Blog post for LWIMIQ Feb 6, 2017

### PR DESCRIPTION
**Pull request by djberg96**
See title.

### CODE REVIEW

**Comment by JPrause** [comment]
@djberg96 thanks for your submission.
@syncrou @jrafanie @imtayadeway can you review this week's blog entry.

**Comment by JPrause** [comment]
LGTM 👍

File: site/_posts/2017-02-06-ansible-and-api-and-stuff.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+tags: LWIMIQ
+comments: true
+published: true
+---
+
+Hi all! [Daniel Berger](https://github.com/djberg96) here from the Providers team. This is my first Last Week in ManageIQ post, where I try to distill the week's highlights of ManageIQ development into a nice, digestable format.
+
+## Featured
+
+This past week saw [52][PRs merged last week] merged in ManageIQ core.
```
Comment (syncrou) [inline site/_posts/2017-02-06-ansible-and-api-and-stuff.md]: Maybe 52 'PRs' merged vs 52 merged.

File: site/_posts/2017-02-06-ansible-and-api-and-stuff.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Improved
+
+This past week saw several improvements for Ansible, including the addition of Ansible Tower provider mixin by [Nick](https://github.com/ManageIQ/manageiq/pull/13733), while Brandon added an [automation manager type](https://github.com/ManageIQ/manageiq/pull/13720) as well as [job templates](https://github.com/ManageIQ/manageiq/pull/13495). Last week also saw Marcel add [graph refresh](https://github.com/ManageIQ/manageiq/pull/13585) for Ansible.
+
+### Everything's better in bulk.
+
+Meanwhile, Tim and Jillian were busy making improvements API side. Among other PR's, Jillian added the ability to [tag unassignments in bulk](https://github.com/ManageIQ/manageiq/pull/13712), as well as the ability to [tag multiple services in bulk](https://github.com/ManageIQ/manageiq/pull/13698). Meanwhile, Tim added [bulk delete for snapshots](https://github.com/ManageIQ/manageiq/pull/13711) and [nicer messages](https://github.com/ManageIQ/manageiq/pull/13710).
```
Comment (syncrou) [inline site/_posts/2017-02-06-ansible-and-api-and-stuff.md]: Possibly 'on the API side'?

File: site/_posts/2017-02-06-ansible-and-api-and-stuff.md (hunk 1 above)
Comment (djberg96) [inline site/_posts/2017-02-06-ansible-and-api-and-stuff.md]: @syncrou Agreed, will fix.

File: site/_posts/2017-02-06-ansible-and-api-and-stuff.md (hunk 2 above)
Comment (djberg96) [inline site/_posts/2017-02-06-ansible-and-api-and-stuff.md]: @syncrou Yep, I'll fix it.

File: site/_posts/2017-02-06-ansible-and-api-and-stuff.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Improved
+
+This past week saw several improvements for Ansible, including the addition of Ansible Tower provider mixin by [Nick](https://github.com/ManageIQ/manageiq/pull/13733), while Brandon added an [automation manager type](https://github.com/ManageIQ/manageiq/pull/13720) as well as [job templates](https://github.com/ManageIQ/manageiq/pull/13495). Last week also saw Marcel add [graph refresh](https://github.com/ManageIQ/manageiq/pull/13585) for Ansible.
+
+### Everything's better in bulk.
+
+Meanwhile, Tim and Jillian were busy making improvements on the API side. Among other PR's, Jillian added the ability to [tag unassignments in bulk](https://github.com/ManageIQ/manageiq/pull/13712), as well as the ability to [tag multiple services in bulk](https://github.com/ManageIQ/manageiq/pull/13698). Meanwhile, Tim added [bulk delete for snapshots](https://github.com/ManageIQ/manageiq/pull/13711) and [nicer messages](https://github.com/ManageIQ/manageiq/pull/13710).
```
Comment (imtayadeway) [inline site/_posts/2017-02-06-ansible-and-api-and-stuff.md]: Wondering if this should say "unassign tags in bulk" and/or "assign tags in bulk" - as Jillian added bulk stuff to all the things :tm:

**Comment by imtayadeway** [comment]
@djberg96 looks great! Just a question above about wording for bulk stuff, otherwise :shipit:

**Comment by syncrou** [comment]
@djberg96 👍 Looks great to me!

File: site/_posts/2017-02-06-ansible-and-api-and-stuff.md (hunk 3 above)
Comment (djberg96) [inline site/_posts/2017-02-06-ansible-and-api-and-stuff.md]: @imtayadeway Agreed.

**Comment by JPrause** [comment]
@djberg96 am I correct in assuming that all edits are done?

**Comment by djberg96** [comment]
@JPrause Yes.

**Comment by JPrause** [comment]
Thanks @djberg96 
@hayesr or @cybette when you have a chance, can you merge this week's blog entry.

**Pull request merged**